build/webpack/webpack.client.config.js

```diff
@@ -18,21 +18,25 @@ module.exports = {
     output: {
         path: path.join(constants.ExtensionRootDir, 'out', 'client_renderer'),
         filename: '[name].js',
-        chunkFilename: `[name].bundle.js`
+        chunkFilename: `[name].bundle.js`,
+        libraryTarget: 'module'
     },
+    experiments: {
+        outputModule: true
+    },
+    target: 'node',
     mode: isProdBuild ? 'production' : 'development',
     devtool: isProdBuild ? 'source-map' : 'inline-source-map',
-    node: {
-        fs: 'empty'
-    },
     plugins: [
-        new FixDefaultImportPlugin(),
         new ForkTsCheckerWebpackPlugin({
             checkSyntacticErrors: true,
             tsconfig: configFileName,
             reportFiles: ['src/client/**/*.{ts,tsx}'],
             memoryLimit: 9096
         }),
+        new DefinePlugin({
```

Comment:
Glad that Conner called this out this change specifically in the issue, would have bit me otherwise.